packages/next/src/client/components/app-router.tsx

```diff
@@ -1,68 +1,70 @@
 'use client'
 
 import React, {
+  startTransition,
   use,
-  useEffect,
-  useMemo,
   useCallback,
-  startTransition,
-  useInsertionEffect,
   useDeferredValue,
+  useEffect,
+  useInsertionEffect,
+  useMemo,
 } from 'react'
-import {
-  AppRouterContext,
-  LayoutRouterContext,
-  GlobalLayoutRouterContext,
-} from '../../shared/lib/app-router-context.shared-runtime'
+import type { FlightRouterState } from '../../server/app-render/types'
```

Comment:
Why are all these imports changing? Sorry to comment on something so trivial but it bothers me when PRs change unrelated lines for seemingly no good reason. Like if you want to reorder the imports then it should be a separate PR.